RecoParticleFlow/PFTracking/src/PFTrackAlgoTools.cc

```diff
@@ -36,7 +36,7 @@ namespace PFTrackAlgoTools {
     case reco::TrackBase::hltIterX:
       return  cuts[0];
     default:
-      return hltIterativeTracking ? cuts[6]:cuts[0];
+      return hltIterativeTracking ? cuts.at(5):cuts[0];
```

Comment:
I find this way of passing and applying cuts (in a vector, with a magic way of picking the index) to defeat the whole purpose of having variable names and a configuration language.